tests/functional/metadata/mongodb/putObject.spec.js

```diff
@@ -0,0 +1,442 @@
+            _name: BUCKET_NAME,
+            _owner: 'testowner',
+            _ownerDisplayName: 'testdisplayname',
+            _creationDate: new Date().toJSON(),
+            _acl: {
+                Canned: 'private',
+                FULL_CONTROL: [],
+                WRITE: [],
+                WRITE_ACP: [],
+                READ: [],
+                READ_ACP: [],
+            },
+            _mdBucketModelVersion: 10,
+            _transient: false,
+            _deleted: false,
+            _serverSideEncryption: null,
+            _versioningConfiguration: null,
+            _locationConstraint: 'us-east-1',
+            _readLocationConstraint: null,
+            _cors: null,
+            _replicationConfiguration: null,
+            _lifecycleConfiguration: null,
+            _uid: '',
+            _isNFS: null,
+            ingestion: null,
+        });
+        metadata.createBucket(BUCKET_NAME, bucketMD, logger, err => {
+            if (err) {
+                return done(err);
+            }
+            collection = metadata.client.getCollection(BUCKET_NAME);
+            return done();
+        });
+    });
+
+    afterEach(done => {
+        metadata.deleteBucket(BUCKET_NAME, logger, done);
+    });
+
+    it('Should put a new non versionned object', done => {
+        const bucketName = BUCKET_NAME;
+        const objName = OBJECT_NAME;
+        const objVal = {
+            key: OBJECT_NAME,
+            versionId: 'null',
+            updated: false,
+        };
+        const params = {
+            versioning: null,
+            versionId: null,
+            repairMaster: null,
+        };
+        async.waterfall([
+            next => metadata.putObjectMD(bucketName, objName, objVal, params, logger, err => {
+                if (err) {
+                    return next(err);
+                }
+                return next();
+            }),
+            next => getObject(objName, null, BUCKET_VFORMAT_V0, (err, object) => {
+                assert.deepStrictEqual(err, null);
+                assert.strictEqual(object.key, OBJECT_NAME);
+                return next();
+            }),
+            // When versionning not active only one document is created (master)
+            next => getObjectCount((err, count) => {
+                assert.deepStrictEqual(err, null);
+                assert.strictEqual(count, 1);
+                return next();
+            }),
+        ], done);
+    });
+
+    it('Should update the metadata', done => {
+        const bucketName = BUCKET_NAME;
+        const objName = OBJECT_NAME;
+        const objVal = {
+            key: OBJECT_NAME,
+            versionId: 'null',
+            updated: false,
+        };
+        const params = {
+            versioning: null,
+            versionId: null,
+            repairMaster: null,
+        };
+        async.waterfall([
+            next => metadata.putObjectMD(bucketName, objName, objVal, params, logger, err => {
```

Comment:
Could we not just use 'next' as last argument?

```suggestion
            next => metadata.putObjectMD(bucketName, objName, objVal, params, logger, next)
```

Reply:
fixed in https://github.com/scality/Arsenal/pull/1729/commits/a6f48b9f342344e6df5ca2e3b7a2d03d8737fbba